index.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
 	development: ['> 1%', 'last 2 versions', 'not IE 11', 'not dead'],
-	production: ['> 0.5%', 'last 3 major versions', 'not IE 11', 'not dead'],
+	production: ['> 0.5%', 'last 3 major versions', 'last 4 years', 'not IE 11', 'not dead'],
```

Comment:
Can we drop `last 3 major versions`?

Reply:
I'll do some testing, I think probably yes.